arch/xtensa/src/esp32/esp32_spiflash.c

```diff
@@ -202,6 +198,10 @@ static struct esp32_spiflash_s g_esp32_spiflash1 =
   .dummies = g_rom_spiflash_dummy_len_plus
 };
 
+/* Enxusre exculisve access to the driver */
+
+static sem_t s_exclsem = SEM_INITIALIZER(1);
```

Comment:
Please change its name to 'g_spifexclsem' to follow NuttX standard of declaring global variables starting with "g_"